controllers/util/util.go

```diff
@@ -5,6 +5,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const onboardingValidationPrivateKeySecretName = "onboarding-private-key"
```

Comment:
could you pls move this const to the file where it is being used?

Reply:
ack